src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -79,7 +79,7 @@ function MdProgressCircularDirective($$rAF, $window, $mdProgressCircular, $mdThe
         var mode = hasValue ? MODE_DETERMINATE : MODE_INDETERMINATE;
         var info = "Auto-adding the missing md-mode='{0}' to the ProgressCircular element";
 
-          // $log.debug( $mdUtil.supplant(info, [mode]) );
+        $log.debug( $mdUtil.supplant(info, [mode]) );
```

Comment:
I remember there being a problem with $log not being present sometimes. @jelbourn Do you remember what this was?